doc/user/zebra.rst

```diff
@@ -521,6 +521,94 @@ entered within the VRF context the static routes are created in the VRF.
    show ip table vrf r1-cust1 table 43
 
 
+.. _zebra-mpls:
+
+MPLS Commands
+=============
+
+You can configure static mpls entries in zebra. Basically, handling MPLS
+consists of popping, swapping or pushing labels to IP packets.
+
+MPLS Acronyms
+-------------
+
+:abbr:`LSR (Labeled Switch Router)`
+   Networking devices handling labels used to forward traffic between and through
+   them.
+
+:abbr:`LER (Labeled Edge Router)`
+   A Labeled edge router is located at the edge of an MPLS network, generally
+   between an IP network and an MPLS network.
+
+MPLS Push Action
+----------------
+
+The push action is generally used for LER devices, which want to encapsulate
+all traffic for a wished destination into an MPLS label. This action is stored
+in routing entry, and can be configured like a route:
+
+.. index:: [no] ip route NETWORK MASK GATEWAY|INTERFACE label LABEL
+.. clicmd:: [no] ip route NETWORK MASK GATEWAY|INTERFACE label LABEL
+
+   NETWORK ans MASK stand for the IP prefix entry to be added as static
+   route entry.
+   GATEWAY is the gateway IP address to reach, in order to reach the prefix.
+   INTERFACE is the interface behind which the prefix is located.
+   LABEL is the MPLS label to use to reach the prefix abovementioned.
+
+  You can check that the static entry is stored in the zebra RIB database, by
+  looking at the presence of the entry.
+
+::
+
+   zebra(configure)# ip route 1.1.1.1/32 10.0.1.1 label 777
+   zebra# show ip route
+   Codes: K - kernel route, C - connected, S - static, R - RIP,
+   O - OSPF, I - IS-IS, B - BGP, E - EIGRP, N - NHRP,
+   T - Table, v - VNC, V - VNC-Direct, A - Babel, D - SHARP,
+   F - PBR,
+   > - selected route, * - FIB route
+
+   S>* 1.1.1.1/32 [1/0] via 10.0.1.1, r2-eth0, label 777, 00:39:42
```

Comment:
If this is part of the command documentation, everything from the `::` down to the line should be indented 3 spaces.

Reply:
I don't understand your remark, since it seems all is indented 3 spaces.
note that the first line between '::' and the first comment is one empty line.

 so I was wondering what was wrong?

Reply:
I said `everything from the :: down to this line`.

So what I'm saying is change it to this:
```
. index:: [no] ip route NETWORK MASK GATEWAY|INTERFACE label LABEL
.. clicmd:: [no] ip route NETWORK MASK GATEWAY|INTERFACE label LABEL

   NETWORK ans MASK stand for the IP prefix entry to be added as static
   route entry.
   GATEWAY is the gateway IP address to reach, in order to reach the prefix.
   INTERFACE is the interface behind which the prefix is located.
   LABEL is the MPLS label to use to reach the prefix abovementioned.

   You can check that the static entry is stored in the zebra RIB database, by
   looking at the presence of the entry.

   ::

      zebra(configure)# ip route 1.1.1.1/32 10.0.1.1 label 777
      zebra# show ip route
      Codes: K - kernel route, C - connected, S - static, R - RIP,
      O - OSPF, I - IS-IS, B - BGP, E - EIGRP, N - NHRP,
      T - Table, v - VNC, V - VNC-Direct, A - Babel, D - SHARP,
      F - PBR,
      > - selected route, * - FIB route

      S>* 1.1.1.1/32 [1/0] via 10.0.1.1, r2-eth0, label 777, 00:39:42
```

Reply:
ok. thanks.
I did the changes, then. please review.